libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/internal/navigation/MapboxNavigationNotificationManeuverInfoTest.java

```diff
@@ -0,0 +1,689 @@
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_MODIFIER_STRAIGHT;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_MODIFIER_UTURN;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_ARRIVE;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_DEPART;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_END_OF_ROAD;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_FORK;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_MERGE;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_OFF_RAMP;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_ROUNDABOUT;
+import static com.mapbox.services.android.navigation.v5.navigation.NavigationConstants.STEP_MANEUVER_TYPE_TURN;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyFloat;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings("KotlinInternalInJava")
+public class MapboxNavigationNotificationManeuverInfoTest extends BaseTest {
+
+    private static final String DIRECTIONS_ROUTE_FIXTURE = "directions_v5_precision_6.json";
+    private DirectionsRoute route;
+
+    @Before
+    public void setUp() throws Exception {
+        final String json = loadJsonFixture(DIRECTIONS_ROUTE_FIXTURE);
+        Gson gson = new GsonBuilder()
+                .registerTypeAdapterFactory(DirectionsAdapterFactory.create()).create();
+        DirectionsResponse response = gson.fromJson(json, DirectionsResponse.class);
+        route = response.routes().get(0);
+    }
+
+    @Test
+    public void checksManeuverInfoIsRetrievedForTurnUturnRightDrivingSide() {
+        MapboxNavigationNotification mapboxNavigationNotification = buildMapboxNavigationNotification();
+        LegStep step = buildLegStep(NavigationConstants.STEP_MANEUVER_TYPE_TURN, NavigationConstants.STEP_MANEUVER_MODIFIER_UTURN,
+                NavigationConstants.STEP_MANEUVER_MODIFIER_RIGHT);
+        RouteProgress mockedRouteProgress = buildMockedRouteProgress(step);
+
+        mapboxNavigationNotification.updateNotificationViews(mockedRouteProgress);
+
+        assertEquals(NavigationConstants.STEP_MANEUVER_TYPE_TURN, mapboxNavigationNotification.getCurrentManeuverType());
```

Comment:
most of this file change are just rename and update the assertions from asserting maneuverResourceId to asserting maneuverType.